src/main/minio.js

```diff
@@ -40,6 +40,8 @@ import { isValidPrefix, isValidEndpoint, isValidBucketName,
 
 import { signV4, presignSignatureV4, postPresignSignatureV4 } from './signing.js';
 
+import { isValidBucketPolicy } from './bucket-policy'
```

Comment:
Thanks, I forgot about that one — I had that in for the `setBucketPolicy` stuff, but I'll be adding that to this commit momentarily so it'll be all bueno.